frontend/coprs_frontend/tests/request_test_api.py

```diff
@@ -424,9 +434,9 @@ def fail_source_build(self, build_id):
         assert self.update(form_data).status_code == 200
 
 
-    def finish_build(self, build_id, package_name=None, pkg_version="1"):
+    def finish_srpm_and_import(self, build_id, package_name=None, pkg_version="1"):
         """
-        Given the build_id, finish the build with succeeded state
+        Given the build_id, finish the source build (but don't import)
```

Comment:
`finish_srpm_and_import` but `finish the source build (but don't import)`. I am confused :smile: